.github/workflows/main.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       matrix:
         config:
           - name: Linux GCC
-            os: ubuntu-latest
+            os: ubuntu-24.04
```

Comment:
Also for the cppcheck task, please.

Reply:
Looks like very few VMs are available, we'll have to wait.